scipy/special/cephes/igam.c

```diff
@@ -164,9 +170,15 @@ double igamc(double a, double x)
 {
     double absxma_a;
 
-    if (x < 0 || a <= 0) {
+    if (x < 0 || a < 0) {
 	sf_error("gammaincc", SF_ERROR_DOMAIN, NULL);
 	return NPY_NAN;
+    } else if (a == 0) {
```

Comment:
As above about moving the `x == 0` branch up.

Reply:
done